proposals/0003-dynamic-compute-unit-base-fee.txt

```diff
@@ -0,0 +1,164 @@
+---
+simd: '0003'
```

Comment:
Update the SIMD number to `0004` here as per the SIMD-0001 numbering [spec](proposals/0003-dynamic-compute-unit-base-fee.txt).

Also, please update the filename to `0004-dynamic-compute-unit-base-fee.md`